css/style.css

```diff
@@ -0,0 +1,15 @@
+#image {
+margin: 0 auto;
+max-width:100%;
+height: auto;
+display: block;
+}
+body{
+    margin: 0;
+}
+#main {
+ display: flex;
+ justify-content: center;
+ align-items: center;
+ flex-direction: column;
+}
```

Comment:
I think it would be better to use  Linters  for  this project to help test stylelint errors in this file
you can use this to ensure that you have [no lint errors ](https://github.com/microverseinc/linters-config)

Reply:
- [ ] Weldone advising the student to use linters, I advise you comment on the appropriate code or in your summary. This helps the student understand better where changes are requested 🤗